### PR TITLE
Feat: Cta section for flutter

### PR DESCRIPTION
**Pull request by cp-ashish-k**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [ ] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [ ] Tablet (chrome, firefox)

### CODE REVIEW

**Comment by cp-ashish-k** [comment]
Cta Section for Flutter development page

- Lighthouse score for desktop view
image

- Lighthouse score for mobile view
 
image

**Comment by cp-ashish-k** [comment]
Design Let's code your success story together! CTA section
 
-  LightHouse score for mobile
Screenshot from 2023-11-07 10-08-34

- Lighthouse Score for desktop
Screenshot from 2023-11-06 17-44-58

- Video of the UI
[Screencast from 07-11-23 11:26:19 AM IST.webm](https://github.com/canopas/canopas-website/assets/140039768/2cd674c4-4824-4d43-b03b-c12d2037cbfa)

File: nuxt-frontend/components/flutter-app-development/CtaSection.vue [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          />
+          <div
+            class="flex flex-col gap-4 lg:gap-6 lg:text-left lg:flex-1 lg:my-auto"
+          >
+            <div>
+              <p class="header-1">Let's code your success story together!</p>
+            </div>
+            <div>
+              <span class="sub-h1-regular">
+                Join hands with us for your next app endeavour.
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/CtaSection.vue:36]: use text color and opacity here same as figma

File: nuxt-frontend/components/flutter-app-development/CtaSection.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            :src="hex"
+            class="flex-1 w-full h-full object-cover hidden lg:block"
+            alt="flutter-hex-image"
+            loading="lazy"
+          />
+          <div
+            class="flex flex-col gap-4 lg:gap-6 lg:text-left lg:flex-1 lg:my-auto"
+          >
+            <div>
+              <p class="header-1">Let's code your success story together!</p>
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/CtaSection.vue:32]: here also you have not considered text color and opacity

File: nuxt-frontend/components/flutter-app-development/CtaSection.vue [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<template>
+  <section>
+    <div class="relative h-[40vh] lg:h-[65vh] 2xl:h-[670px]">
+      <img
+        :src="background400"
+        :srcset="`${background400} 400w, ${background800} 800w`"
+        class="h-full w-full object-cover opacity-20 lg:hidden"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/CtaSection.vue:7]: this image is cutting after 446 width

File: nuxt-frontend/components/flutter-app-development/CtaSection.vue [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        alt="flutter-cta"
+        loading="lazy"
+      />
+      <img
+        :src="background1200"
+        :srcset="`${background1200} 1200w, ${background2400} 2400w`"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/CtaSection.vue:13]: i think in this background images for both mobile and desktop we should use blur images given in figma

File: nuxt-frontend/components/flutter-app-development/CtaSection.vue [hunk 5]
```diff
@@ -0,0 +1,76 @@
+<template>
+  <section>
+    <div class="relative h-[40vh] lg:h-[65vh] 2xl:h-[670px]">
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/CtaSection.vue:3]: compare this cta's height with other page cta its too much bigger cta should be of same height everywhere

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-dharti-r — APPROVED** [review]

**Pull request merged**